server/src/main/java/jass/server/util/GameUtil.java

```diff
@@ -332,6 +367,206 @@ public void onPlayedCard(final PlayCardData data) throws InterruptedException {
         }
     }
 
+    ////////////////////////////////////////////////////////////////////////////
+    // Figure out the points for cards.
+    ////////////////////////////////////////////////////////////////////////////
+
+    /**
+     * @param cards  All cards
+     * @param trumpf The trump suit to check
+     *
+     * @return Returns the points of all cards together.
+     *
+     * @author Manuele Vaccari
+     */
+    private static int calculateCardPointsTrumpf(final ArrayList<CardEntity> cards, final Card.Suit trumpf) {
+        // Convert suit enum to database entity
+        SuitEntity trumpfSuit = SuitRepository.getSingleton(null).getByName(trumpf.toString().toLowerCase());
+        assert trumpfSuit != null;
+
+        int points = 0;
+        for (CardEntity card : cards) {
+            if (card.getSuit().equals(trumpfSuit)) {
+                if (card.getRank().getKey().equals("jack")) {
+                    points += 21;
```

Comment:
this needed to be 21 points. But I directly changed.

Reply:
it is 20 points
https://www.pagat.com/de/jass/swjass.html
https://www.swisslos.ch/de/jass/informationen/jass-regeln/jass-grundlagen.html